docs/manual/howtos/listener.rst

```diff
@@ -12,12 +12,12 @@
 Listeners
 =========
 
-Listeners allow the code to react to changes in state of DDS entities like readers, writers, etc.
+Listeners enable the code to react to changes in state of DDS entities such as readers, writers, and so on.
 CycloneDDS-CXX implements different listeners for different entities.
 Some types' listeners inherit from other types' listeners, allowing the listener of one type to react to changes in state on subordinate entities.
-E.G.: the listener for a DomainParticipant also implements the functionality of the listeners for Publishers, Subscribers and Topics.
-By using this functionality, the user can avoid large amounts of boilerplate code, since the listener will have to be implemented just one time.
-The listener can then be propagated down to the Publisher and from the Publisher to a DataWriter for instance.
+For example, the listener for a DomainParticipant also implements the functionality of the listeners for Publishers, Subscribers and Topics.
+By using this functionality, the user can avoid large amounts of boilerplate code, because the listener is implemented just the one time.
```

Comment:
"because the listener is implemented just once"
I think it is important to have the point here that the user is the one doing the implementing of the listener class, not CycloneDDS, which it now seems like.

So maybe rewrite to:
"for example, by implementing the required callback functions for Topics and Subscribers in a DomainParticipantListener, this listener only needs to be set once at the DomainParticipant, and the Topic and Subscriber will then propagate the events to this Listener."

Reply:
Agreed. Changed to your re-write.